package-lock.json

```diff
@@ -0,0 +1,13093 @@
+{
+  "name": "notes",
+  "version": "2.2.0dev",
+  "lockfileVersion": 1,
```

Comment:
if we are using a package-lock, could we please specify the npm version required for this project in package.json, see example at https://docs.npmjs.com/files/package.json#engines